qcommon/src/org/qcmg/common/vcf/VcfUtils.java

```diff
@@ -1504,22 +1504,31 @@ public static String getConfidence(VcfRecord rec) {
 	 * @param minCoverage
 	 * @return
 	 */
-	public static boolean mutationInNormal(int altCount, int totalReadCount, float percentage, int maxCoverage) {
-		if (altCount == 0) {
+	public static boolean mutationInNormal(int altCount, int totalReadCount, float percentage, int minValue) {
+		if (altCount == 0 || altCount < minValue) {
 			return false;
 		}
 		/*
-		 * if altCount is greater than the maxCoverage
+		 * Need to get the max(minValue, percentage * totalReadCount) and see if the alt count is larger than that
 		 */
-		if (altCount >= maxCoverage) {
-			return true;
-		}
-		/*
-		 * calculate percentage
-		 */
-		float passingCount = totalReadCount > 0 ? (((float)totalReadCount / 100) * percentage) : 0;
-		return altCount >= passingCount;
+		return altCount >= Math.max(minValue, (totalReadCount > 0 ? (((float)totalReadCount / 100) * percentage) : 0));
 	}
+//	public static boolean mutationInNormal(int altCount, int totalReadCount, float percentage, int maxCoverage) {
+//		if (altCount == 0) {
+//			return false;
+//		}
+//		/*
+//		 * if altCount is greater than the maxCoverage
+//		 */
+//		if (altCount >= maxCoverage) {
+//			return true;
+//		}
+//		/*
+//		 * calculate percentage
+//		 */
+//		float passingCount = totalReadCount > 0 ? (((float)totalReadCount / 100) * percentage) : 0;
+//		return altCount >= passingCount;
+//	}
```

Comment:
rm unless keeping for archaeological reasons

Reply:
gone